recipe/meta.yaml

```diff
@@ -9,22 +12,35 @@ package:
 source:
   url: https://github.com/rapidsai/ptxcompiler/archive/refs/tags/{{ version }}.tar.gz
   sha256: 469644ac7d1d37ab82b7213d3aae90820c3d840aa483967b006554501fe725ac
+  patches:
+    ##########################################################
+    # Add `$CUDA_HOME/lib` to library search path.           #
+    #                                                        #
+    # xref: https://github.com/rapidsai/ptxcompiler/pull/40  #
+    ##########################################################
+    - PR_40.patch
```

Comment:
Ah! We were only searching `$CUDA_HOME/lib64` and not `$CUDA_HOME/lib`. Submitted upstream PR ( https://github.com/rapidsai/ptxcompiler/pull/40 ) to fix. For now added as a patch here